client/flutter/lib/main.dart

```diff
@@ -83,36 +83,8 @@ class _MyAppState extends State<MyApp> {
     // onMessage: Fires when app is foreground
```

Comment:
Can you move these comments to where `onMessage`, `onLaunch`, and `onResume` are now defined?